### PR TITLE
New Source Cart (#4588): Documentation

### PR DESCRIPTION
**Pull request by vovavovavovavova**
## What
Documentation for #4588 (extends https://github.com/airbytehq/airbyte/pull/4785 )

## How
*Describe the solution*

## Recommended reading order
1. `x.java`
2. `y.python`

## Pre-merge Checklist
Expand the checklist which is relevant for this PR. 

<details><summary> <strong> Connector checklist </strong> </summary>
<p>

- [ ] Issue acceptance criteria met
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [ ] Secrets are annotated with `airbyte_secret` in the connector's spec
- [ ] Credentials added to Github CI if needed and not already present. [instructions for injecting secrets into CI](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#using-credentials-in-ci). 
- [ ] Unit & integration tests added as appropriate (and are passing)
    * Community members: please provide proof of this succeeding locally e.g: screenshot or copy-paste acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] `/test connector=connectors/<name>` command as documented [here](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#updating-an-existing-connector) is passing. 
    * Community members can skip this, Airbyters will run this for you. 
- [ ] Code reviews completed
- [ ] Documentation updated 
    - [ ] `README.md`
    - [ ] `docs/SUMMARY.md` if it's a new connector
    - [ ] Created or updated reference docs in `docs/integrations/<source or destination>/<name>`.
    - [ ] Changelog in the appropriate page in `docs/integrations/...`. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [ ] `docs/integrations/README.md` contains a reference to the new connector
    - [ ] Build status added to [build page](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/builds.md)
- [ ] Build is successful
- [ ] Connector version bumped like described [here](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#updating-a-connector)
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#updating-a-connector)
- [ ] No major blockers
- [ ] PR merged into master branch
- [ ] Follow up tickets have been created
- [ ] Associated tickets have been closed & stakeholders notified
</p>
</details>

<details><summary> <strong> Connector Generator checklist </strong> </summary>
<p>
   
- [ ] Issue acceptance criteria met
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [ ] If adding a new generator, add it to the [list of scaffold modules being tested](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/generator/build.gradle#L41)
- [ ] The generator test modules (all connectors with `-scaffold` in their name) have been updated with the latest scaffold by running `./gradlew :airbyte-integrations:connector-templates:generator:testScaffoldTemplates` then checking in your changes
- [ ] Documentation which references the generator is updated as needed.
</p>
</details>

### CODE REVIEW

**Review by sherifnada — APPROVED** [review]

File: docs/integrations/sources/cart.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Cart
+
+## Sync overview
+
+This source can sync data for the [Cart API](https://developers.cart.com/docs/rest-api/docs/README.md). All streams we support are Incremental sync.
```
Comment (keu) [inline docs/integrations/sources/cart.md:5]:
```suggestion
This source can sync data for the [Cart API](https://developers.cart.com/docs/rest-api/docs/README.md). It supports both Full Refresh and Incremental sync for all streams.
```

Comment (keu) [inline docs/integrations/sources/cart.md:5]:
```suggestion
This source can sync data for the [Cart API](https://developers.cart.com/docs/rest-api/docs/README.md). It supports both Full Refresh and Incremental sync for all streams.
```

File: docs/integrations/sources/cart.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+You can choose if this connector will copy only the new or updated data, or all rows in the tables and columns you set up for replication, every time a sync is run.
+
+### Output schema
+
+This Source is capable of syncing the following core Streams:
+
+* [CustomersCart](https://developers.cart.com/docs/rest-api/restapi.json/paths/~1customers/get) (Incremental)
+* [Orders](https://developers.cart.com/docs/rest-api/restapi.json/paths/~1orders/get) (Incremental)
+* [OrderPayments](https://developers.cart.com/docs/rest-api/restapi.json/paths/~1order_payments/get) (Incremental)
+* [Products](https://developers.cart.com/docs/rest-api/restapi.json/paths/~1products/get) (Incremental)
```
Comment (keu) [inline docs/integrations/sources/cart.md:15]:
```suggestion
* [CustomersCart](https://developers.cart.com/docs/rest-api/restapi.json/paths/~1customers/get)
* [Orders](https://developers.cart.com/docs/rest-api/restapi.json/paths/~1orders/get)
* [OrderPayments](https://developers.cart.com/docs/rest-api/restapi.json/paths/~1order_payments/get)
* [Products](https://developers.cart.com/docs/rest-api/restapi.json/paths/~1products/get)
```

File: docs/integrations/sources/cart.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+| Feature | Supported?\(Yes/No\) | Notes |
+| :--- | :--- | :--- |
+| Full Refresh Sync | Yes |  |
+| Incremental Sync | Yes |  |
+| Namespaces | No |  |
+
+### Performance considerations
+
+The Cart api has some request limitation. See [this](https://developers.cart.com/docs/rest-api/docs/README.md#rate-limiting) .
```
Comment (keu) [inline docs/integrations/sources/cart.md:36]: The connector is restricted by normal Cart [requests limitation](https://developers.cart.com/docs/rest-api/docs/README.md#rate-limiting).

The Cart connector should not run into Cart API limitations under normal usage.
Please [create an issue](https://github.com/airbytehq/airbyte/issues) if you see any rate limit issues that are not automatically retried successfully.

File: docs/integrations/sources/cart.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+## Getting started
+
+### Requirements
+
+* Cart access_token
```
Comment (keu) [inline docs/integrations/sources/cart.md:42]: https://developers.cart.com/docs/rest-api/docs/authentication.md#getting-an-access-token
```suggestion
* AmeriCommerce account
* Admin access
* Access Token
```

File: docs/integrations/sources/cart.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+### Setup guide
+
+Please follow these [steps](https://developers.cart.com/docs/rest-api/docs/README.md#setup) to obtain the access token for your account.
```
Comment (keu) [inline docs/integrations/sources/cart.md:46]:
```suggestion
Please follow these [steps](https://developers.cart.com/docs/rest-api/docs/README.md#setup) to obtain Access Token for your account. 
```

**Review by keu — APPROVED** [review]
just few comments

**Pull request merged**